plugins/mcreator-localization/lang/texts.properties

```diff
@@ -916,18 +917,18 @@ elementgui.armor.advanced_body=Advanced body settings
 elementgui.armor.advanced_leggings=Advanced leggings settings
 elementgui.armor.advanced_boots=Advanced boots settings
 elementgui.armor.supported_java=<html>Model: <br><small>Supported: JAVA
-elementgui.armor.texture=<html><small>Texture: 
+elementgui.armor.texture=<html><small>Texture:

```

Comment:
I'm stopping there but there are a lot of lines where you deleted the space at the end of the line for no reasons. That's why you throught nothing was changed in the lang file.